.github/workflows/pull_request.yml

```diff
@@ -34,7 +34,7 @@ jobs:
 
       - name: Install Sphinx
         run: |
-          pip install --user --upgrade --upgrade-strategy eager setuptools sphinx sphinx-rtd-theme restructuredtext_lint rstcheck pygments m2r2
+          pip install --user --upgrade --upgrade-strategy eager docutils==0.16 setuptools sphinx sphinx-rtd-theme restructuredtext_lint rstcheck pygments m2r2
```

Comment:
Please add a comment here explaining why docutils version is selected and pointing to the m2r2 issue.